frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -766,6 +769,41 @@ function ReaderFooter:addToMainMenu(menu_items)
                     self:refreshFooter(true, true)
                 end,
             },
+            {
+                text_func = function()
+                    return T(_("Font size (%1)"), self.settings.text_font_size)
+                end,
+                separator = true,
+                callback = function(touchmenu_instance)
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local font_size = self.settings.text_font_size
+                    local items_font = SpinWidget:new{
+                        width = Screen:getWidth() * 0.6,
+                        value = font_size,
+                        value_min = 10,
+                        value_max = 18,
```

Comment:
Is a max value technically required for this widget? (Not in a position to test atm.) See https://www.mobileread.com/forums/showthread.php?p=3946432#post3946432

Reply:
Yes. max_value is required but I can prepare PR with `value_min = 8` and `value_max = 36`

Reply:
Alright, please do! :+1: